drivers/clock_control/stm32f1x_ll_clock.c

```diff
@@ -67,6 +67,11 @@ void config_pll_init(LL_UTILS_PLLInitTypeDef *pllinit)
 	 */
 	pllinit->Prediv = LL_RCC_PREDIV_DIV_1;
 #endif /* CONFIG_CLOCK_STM32_PLL_XTPRE */
+#else
+#if defined(CONFIG_CLOCK_STM32_PLL_SRC_HSI)
```

Comment:
Seems this applies to both DENSITY and CONNECTIVITY lines, can we put this out of the `#else ( /* CONFIG_SOC_STM32F10X_DENSITY_DEVICE */)` ?
Something like : 
```
#ifdef CONFIG_CLOCK_STM32_PLL_SRC_HSI
/* Your code */
#else
/* Existing PLL HSE code */
#endif
```

Reply:
Certainly.